backend/src/Designer/Services/Implementation/GiteaAPIWrapper/GiteaAPIWrapper.cs

```diff
@@ -204,20 +204,23 @@ public async Task<IList<RepositoryClient.Model.Repository>> GetOrgRepos(string o
         /// <inheritdoc/>
         public async Task<ListviewServiceResource> MapServiceResourceToListViewResource(string org, string repo, ServiceResource serviceResource)
         {
-            ListviewServiceResource listviewResource = new ListviewServiceResource { Identifier = serviceResource.Identifier, Title = serviceResource.Title };
+            ListviewServiceResource listviewResource = new ListviewServiceResource
+            {
+                Identifier = serviceResource.Identifier,
+                Title = serviceResource.Title,
+            };
+
             string resourceFolder = serviceResource.Identifier;
 
-            HttpResponseMessage response = await _httpClient.GetAsync($"repos/{org}/{repo}/contents/{resourceFolder}/{serviceResource.Identifier}_resource.json");
+            HttpResponseMessage fileResponse = await _httpClient.GetAsync($"repos/{org}/{repo}/commits?path={resourceFolder}&stat=false&verification=false&files=false");
```

Comment:
When called with a path query param, the response to this call cannot be paged. Can potentially be slow if a file has very many commits